soccer/planning/planner/PivotPathPlanner.cpp

```diff
@@ -39,12 +39,15 @@ Trajectory PivotPathPlanner::plan(const PlanRequest& request) {
 
     const auto& command = std::get<PivotCommand>(request.motionCommand);
 
-    if (!shouldReplan(command)) {
+    double radius = _pivotRadiusMultiplier->value() * Robot_Radius;
+    auto pivot_point = command.pivotPoint;
+
+    if (cached_pivot_point.has_value() && cached_pivot_point.value().distTo(pivot_point) < Robot_MouthWidth / 2) {
```

Comment:
What's the logic here? Why mouthwidth / 2?

Reply:
The usual thought is that if you did the calculations perfectly, you have about this much wiggle room before you won't hit the ball with the mouth. Replanning for micrometers isn't worth it since replanning caused more performance problems than than being a little off historically.

This measurement falls apart a little when you have to worry about forwards backwards motion, but the thought process was always kinda "meh, there's worse things causing problems"